skyyrose-suite/plugins/skyyrose-market/skills/skyyrose-social-social-listening-plan/SKILL.md

```
@@ -148,14 +147,14 @@ Common variations and misspellings:
 | X/Twitter Advanced Search | Real-time keyword monitoring               | Saved searches for Tier 1 + Tier 5 purchase signals|
 | Reddit search           | Subreddit monitoring (r/streetwear, r/malefashionadvice, r/femalefashionadvice, r/AskSF, r/Oakland) | Weekly keyword scan |
 
-### Google Alerts setup (Tier 1 — set each as a separate alert)
+##### Google Alerts Setup (Tier 1: Separate Alert per Query)
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)